CONTRIBUTING.md

```diff
@@ -4,6 +4,24 @@ This document should concisely express the project development status,
 methodology, and contribution process.  As the community makes progress, we
 should keep this document in sync with reality.
 
+## Submitting a Pull Request (PR)
+
+The following outlines the general rules we follow:
+
+- All PRs must have the appropriate documentation changes made within the
```

Comment:
There needs to be a way to express that we value an up-to-date documentation and that it is important to provide it with the same PR if reasonable, or strive for update soon after PR merge so as not to diverge too much between code and documentation.

Personally, `must`s do trigger my lawyer alter ego (no lawyers in my ancestry, sadly) but `should`s or `recommend`s would seem appropriate, especially when stating rationale: let's not have out-of-date documentation sitting around for too long, without issue open to represent the update. So, fix it soon, or open issue and fix that issue soon :)